Sources/Layout/MessageSizeCalculator.swift

```diff
@@ -95,40 +95,37 @@ open class MessageSizeCalculator: CellSizeCalculator {
         let messageVerticalPadding = messageContainerPadding(for: message).vertical
         let avatarHeight = avatarSize(for: message).height
         let avatarVerticalPosition = avatarPosition(for: message).vertical
-        let accessoryViewHeight = accessoryViewSize(for: message).height
-        let accessoryViewVerticalPadding = accessoryViewPadding(for: message).vertical
-        let accessoryViewTotalHeight = accessoryViewHeight + accessoryViewVerticalPadding
 
         switch avatarVerticalPosition {
         case .messageCenter:
             let totalLabelHeight: CGFloat = cellTopLabelHeight + messageTopLabelHeight
                 + messageContainerHeight + messageVerticalPadding + messageBottomLabelHeight
             let cellHeight = max(avatarHeight, totalLabelHeight)
-            return max(accessoryViewTotalHeight, cellHeight)
+            return cellHeight
```

Comment:
Are we sure this was the correct change? 🤔 Seems like it was correct before

Reply:
Ok, I thought this was one of the things you wanted reverted and if it messed up the height it was a developer error, back to how I originally had it :D